deeppavlov/models/preprocessors/ubuntu_v2_preprocessor.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2018 Neural Networks and Deep Learning lab, MIPT
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+from typing import List, Union, Iterable
+
+from deeppavlov.core.commands.utils import expand_path
+from deeppavlov.core.common.log import get_logger
+from deeppavlov.core.common.registry import register
+from deeppavlov.core.models.component import Component
+from deeppavlov.core.models.estimator import Estimator
+from deeppavlov.core.data.utils import zero_pad_truncate
+
+log = get_logger(__name__)
+
+
+@register('ubuntu_v2_preprocessor')
+class UbuntuV2Preprocessor(Estimator):
```

Comment:
This class could have more general name as it is used with ubuntu v1 as well as with v2.